libs/ui/inputs/src/lib/navigation-button/navigation-button.component.ts

```diff
@@ -6,7 +7,7 @@ import { NgIcon } from '@ng-icons/core'
   templateUrl: './navigation-button.component.html',
   styleUrls: ['./navigation-button.component.css'],
   changeDetection: ChangeDetectionStrategy.OnPush,
-  imports: [NgIcon],
+  imports: [CommonModule, NgIcon],
   standalone: true,
 })
 export class NavigationButtonComponent {
```

Comment:
as mentioned above, I think this component should go; everything should use the same buttons

Reply:
It's gone now:-)